test/bntest.c

```diff
@@ -6,35 +6,23 @@
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
-#include "../e_os.h"
 #include <assert.h>
 #include <errno.h>
 #include <stdio.h>
 #include <string.h>
 #include <ctype.h>
 
-#include "internal/nelem.h"
-#include "internal/numbers.h"
 #include <openssl/bn.h>
 #include <openssl/crypto.h>
 #include <openssl/err.h>
 #include <openssl/rand.h>
+#include "internal/nelem.h"
+#include "internal/numbers.h"
```

Comment:
Should anyone wonder, `stdint.h` is included by at least one of those `<openssl/{whatever}.h>`, and that needs to happen before the inclusion of `"internal/numbers.h"`.  Either way, it makes sense to include out internal headers after the public/system ones, as the former are designed to fix up things that weren't defined by the latter...